src/main/java/com/rybka/view/ExchangeView.java

```diff
@@ -41,9 +50,24 @@ public void showDialog() {
                     total);
 
             hibernateDAO.save(convertedResult);
-            hibernateDAO.showTableRow();
+            var history = hibernateDAO.select();
 
             showExchange(convertedResult);
+
+            switch (userExportChoice) {
+                case 1:
+                    new ConsoleExportService().export(history);
+                    break;
+                case 2:
+                    new CSVExportService().export(history, "test.csv");
+                    break;
+                case 3:
+                    new JSONExportService(new ObjectMapper()).export(history, "test.json");
```

Comment:
Haven't you create instance of ObjectMapper earlier for the Connector service? Why you didn't reuse it?

Reply:
fixed